suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -339,6 +374,20 @@ export const useAddCoinAccount = () => {
         }
     };
 
+    const handleAccountTypeConfirmation = (flowType: AddCoinFlowType) => {
+        if (networkSymbolWithTypeToBeAdded) {
+            // Timeout is needed so AccountTypeDecisionBottomSheet has time to hide otherwise app crashes
+            setTimeout(() => {
+                addCoinAccount({
+                    symbol: networkSymbolWithTypeToBeAdded[0],
+                    accountType: networkSymbolWithTypeToBeAdded[1],
+                    flowType,
+                });
```

Comment:
nit: readability improvement. Feel free to close this.
```suggestion
                const [symbol, accountType] = networkSymbolWithTypeToBeAdded;
                addCoinAccount({
                    symbol,
                    accountType,
                    flowType,
                });
```